.env.example

```
@@ -27,3 +27,7 @@ EXPO_PUBLIC_TENOR_API_KEY=your_tenor_api_key_here
 # Hive Hangouts (required for audio room features)
 EXPO_PUBLIC_HANGOUTS_API_URL=https://hangout-api.3speak.tv
 EXPO_PUBLIC_LIVEKIT_URL=wss://livekit.3speak.tv
+
+# Snapie Translation API (self-hosted LibreTranslate)
+EXPO_PUBLIC_LIBRETRANSLATE_URL=https://translate.snapie.io
+EXPO_PUBLIC_LIBRETRANSLATE_KEY=your_libretranslate_key_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_LIBRETRANSLATE_KEY key should go before the EXPO_PUBLIC_LIBRETRANSLATE_URL key

(UnorderedKey)